src/server/lib/user-features.ts

```diff
@@ -42,3 +60,47 @@ export const setUserFeatureCookies = async ({
 		});
 	}
 };
+
+export const createCookie = ({
+	name,
+	res,
+	daysTillExpiry,
+}: {
+	name: string;
+	res: Response;
+	daysTillExpiry: number;
+}) => {
+	const tmpDate = new Date(Date.now() + daysTillExpiry * 24 * 60 * 60 * 1000);
+	res.cookie(name, tmpDate.getTime().toString(), {
```

Comment:
Is this value coming from somewhere, e.g. DCR?
Can we get an comments explaining where this value is coming from.

Reply:
e.g. as we've done previously for the old way of getting the ad-free cookie:

https://github.com/guardian/gateway/blob/34311cdf8524f247a11841fbf7bdf9568f5829ea/src/server/lib/user-features.ts#L34-L42